understanding/audio-control.html

```diff
@@ -58,294 +56,216 @@
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.4.2:</span>Audio Control (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.4.2:</span>音声の制御 (レベル A)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>If any audio on a Web page plays automatically for more than 3 seconds, either a <a href="#dfn-mechanism">mechanism</a> is available to pause or stop the audio, or a mechanism is available to control audio
-                     volume independently from the overall system volume level.
+                  <p>ウェブページ上にある音声が自動的に再生され、3 秒より長く続く場合、その音声を一時停止又は停止する<a href="#dfn-mechanism">メカニズム</a>、もしくはシステム全体の音量レベルに影響を与えずに音量レベルを調整できるメカニズムが利用できる。
+
                      
                   </p>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>Since any content that does not meet this success criterion can interfere with a user's
-                        ability to use the whole page, all content on the Web page (whether or not it is used
-                        to meet other success criteria) must meet this success criterion. See <a href="#cc5">Conformance Requirement 5: Non-Interference</a>.
+                     <p class="note-title marker">注記</p>
+                     <p>この達成基準を満たさないコンテンツでは、利用者がそのウェブページ全体を使用できない恐れがあるため、ウェブページ上のすべてのコンテンツは他の達成基準を満たすために用いられているか否かにかかわらず、この達成基準を満たさなければならない。<a href="https://waic.jp/translations/WCAG22/#cc5">適合要件 5: 非干渉</a>を参照。
                         
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>A page that plays music or sounds doesn't disrupt people.</dd>
+                  <dt>目標</dt>
+                  <dd>音楽又は音を再生するページでは、人々の邪魔にならないようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>If you play audio content automatically, let people turn it down or off.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>音声コンテンツを自動的に再生する場合は、音量を下げたり再生を止めたりできるようにする。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Sound distracts some people, and also interferes with screen readers.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>音によって気が散ってしまう人がいる。かつ、音はスクリーンリーダーの邪魔にもなる。</dd> 
```

Comment:
ここのandは「そして」でいいんじゃないですかね…
```suggestion
                  <dd>音によって気が散ってしまう人がいる。そして、音はスクリーンリーダーの邪魔にもなる。</dd> 
```